pasta_eln/Extractors/extractor_json.py

```diff
@@ -16,13 +17,17 @@ def use(filePath, recipe='', saveFileName=None):
   """
   # Extractor for fancy instrument
   content = ''
-  with open(filePath,'r', encoding='utf-8') as jsonFile:
-    jsonContent = jsonFile.read()
-    content = json.loads(jsonContent)
-    if not isinstance(content, dict):
-      content= {'content': json.loads(jsonContent)}
+  fileSize = os.stat(filePath).st_size / (1024*1024)  #in MB
```

Comment:
Why do we limit the size of the json? Anything above 0.3 Mb creates some performance problem?

Reply:
Yes, indeed. If people come with super large json, the performance goes down and nobody will look through those multi-MB json anyhow.